docs/developer/order/order-expiration.mdx

```diff
@@ -2,6 +2,9 @@
 title: Order Expiration
 ---
 
+Once an unpaid order is created (via [`checkoutComplete`](api-reference/checkout/mutations/checkout-complete.mdx) or [`orderCreateFromCheckout`](api-reference/orders/mutations/order-create-from-checkout.mdx)), it remains in `UNCONFIRMED` status. If the customer does not complete the payment within `expireOrdersAfter` minutes after that, the status will change to `EXPIRED` and stock allocation will be released.
```

Comment:
Maybe we can add sentence that explain more business context like "Orders can expire to prevent stale or abandoned orders from blocking inventory and cluttering the system."

Reply:
I won't have time on my end to do that change, if it needs to be improved it would be better to be done in a separate PR 🙂